zenodo/modules/records/templates/zenodo_records/box/info.html

```diff
@@ -220,7 +220,7 @@ <h4>
     <dt>{{ _('Custom keywords') }}:</dt>
     <dd class="custom-keywords">
       <ul class="list-unstyled">
-      {%- for custom_field, custom_field_values in record.custom.items() %}
+      {%- for custom_field, custom_field_values in record.custom.items() | sort_custom_keywords %}
```

Comment:
```suggestion
      {%- for custom_field, custom_field_values in record.custom | sort_custom_keywords %}
```
Passing the entire dictionary would allow for the Jinja filter to deal with calling `.items()` or anything else. This is also to make the input of the filter more flexible (theoretically now, the function's signature accepts a single argument of a list of 2-tuples vs. a single dictionary).